include/sway/input/cursor.h

```diff
@@ -67,6 +67,9 @@ struct sway_cursor {
 	bool hidden;
 
 	size_t pressed_button_count;
+
+	bool simulating_pointer;
```

Comment:
Two small nits about this:
* this would be best to keep right after the declarations for `touch_down`, `touch_up` etc. so they're in one group
* `simulating_pointer` is a bit of a misleading name. Since we might be emulating pointer input for both tablet and touch devices _at the same time_, a single field isn't sufficient the way this implies. Tablets now use `cursor->simulated_tool_tip_down`, so a more specific name would be good here.

Reply:
Yeah I wasn't sure about the naming either, so I just went with whatever came to my mind first.
How about simulated_touch_down? To me this sounds like the touch down is being simulated though but it's more in line with the tablet version (which again to me sounds like the tool tip is being simulated not the pointer but that might be just me). I personally would probably prefer simulated_by_touch and simulated_by_tablet.

Reply:
Yeah, I'm not a huge fan of `simulated_tool_tip_down` either, since what's being simulated is "backwards". `cursor->simulated_by_touch/tablet` is a bit better, but it makes it sound like the _cursor_ is being simulated.

Maybe `simulating_pointer_from_touch` and `simulating_pointer_from_tool_tip` (or `simulating_touch_as_pointer` and `simulating_tool_tip_as_pointer`)? A bit of a mouthful, but more "clear" (and only used in one or two places).

Reply:
I went with simulating_pointer_from_touch since I feel it's the clearest and as you said, it's only used a few times.